website/content/options/rnval/_index.md

```diff
@@ -0,0 +1,17 @@
+```
+usage: rnval [-p] [-m MIN] [-M MAX] [-r RISK] [-h]
+```
+
+calculates the value of an option based on ending price estimates. this
+calculates the expected value of the option payoff and then compares it
+to current prices.
+learn more: https://en.wikipedia.org/wiki/Rational_pricing#Delta_hedging
```

Comment:
This is not delta hedging.

Aren't you just calculating the expected value for the option assuming it grew at a risk free rate (i.e https://www.equitymethods.com/articles/real-world-vs-risk-neutral-understanding-the-difference-between-valuation-and-derived-service-period-assumptions/ )?

Reply:
So I would envision an example (similar to something like the link I sent) that walks through the calculation and what the calculation actually means.  Also, unrelated but can this be applied to continuous distributions?

Reply:
Would you like me to just include this link? Also, this could be related to continuous distributions.